bindings/wysiwyg-ffi/src/wysiwyg_composer.udl

```diff
@@ -32,6 +35,12 @@ interface ComposerUpdate {
     sequence<ComposerAction> actions();
 };
 
+dictionary ComposerState {
+    sequence<u16> html;
+    u64 start;
```

Comment:
Why u64? In other places I was using u32 - should I have used u64?

Reply:
`u32` might be enough, I'm used to using Long (64bit signed int) for indexes or sizes, but since this is `u64` it will be twice the size. I used `u64` because the bindings cannot directly translate `usize` as far as I could see and casting `usize` to `u32` could hypothetically truncate the number. But since we'll be using this for text messages I doubt anyone will write a message that long 😅 .

Reply:
Yeah, I think `u64` is probably a better choice everywhere I used `u32` since it has a lossless conversion from `usize`, although it does mean we have a lossy conversion on the way in, so maybe it's no better? (And as you say, in practice we are not expecting numbers to get anywhere near the limits.)